mix.exs

```diff
@@ -43,9 +43,8 @@ defmodule Hammer.Plug.MixProject do
       {:credo, "~> 1.6", only: [:dev, :test]},
       {:dialyxir, "~> 1.1", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.28", only: :dev},
-      {:hammer, "~> 6.0"},
-      {:plug, "~> 1.0"},
-      {:mock, "~> 0.3.0", only: :test}
+      {:hammer, github: "ruslandoga/hammer", branch: "new-ets"},
```

Comment:
Should be changed to `~> 7.0` once https://github.com/ExHammer/hammer/pull/72 or similar is merged.